### DISCUSSION TITLE
Keyboard Controls

### DISCUSSION THREAD

**Discussion by pinchese** [stoatchat/discussions#56] [Feature Suggestions]
as an avid user of my keyboard, i'd really love more keyboard shortuts
such as tab to focus the chatbox, from which i can use the arrow keys to move up and down,
pressing r to reply to the focused message,
and ctrl comma for settings

**Comment by Pupusaru** [comment]
yes please +1

**Comment by pdurbin** [comment]
This is the oldest request for keyboard shortcuts but there have been others:

- https://github.com/orgs/stoatchat/discussions/189
- https://github.com/orgs/stoatchat/discussions/414
- https://github.com/orgs/stoatchat/discussions/771
- https://github.com/orgs/stoatchat/discussions/821
- https://github.com/orgs/stoatchat/discussions/978
- https://github.com/stoatchat/for-desktop/issues/61
